deploy/cephfs/kubernetes/v1.13/csi-cephfsplugin-provisioner.yaml

```diff
@@ -117,3 +137,19 @@ spec:
           emptyDir: {
             medium: "Memory"
           }
+---
+# This is a service to expose the liveness side car
+apiVersion: v1
```

Comment:
this  applies for both cephfs and rbd

Reply:
i can do that but will have to make changes to the existing service. it will end up looking like this

> kind: Service
apiVersion: v1
metadata:
  name: csi-cephfsplugin-provisioner
  labels:
    app: csi-liveness
spec:
  selector:
    app: csi-cephfsplugin-provisioner
  ports:
    - name: dummy
      port: 12345
    - name: http-metrics
      port: 8080
      protocol: TCP
      targetPort: 8081

is that ok?

Reply:
just remove the dummy port . its of no use.